client/src/styles/MilestoneTable.css

```diff
@@ -0,0 +1,25 @@
+.table-wrapper {
+    display: flex;
+    width: 100%;
+    height: 100px;
+    justify-content: center;
+    align-items: center;
+    border-radius: 7px;
+    background-color: #f5f5f5;
+/* #eef4f9 */
```

Comment:
Make sure you take a look through the pr and remove any unused/unwanted code

Reply:
Just to wanted save this color as I like it, cool, nice point